ddspipe_core/include/ddspipe_core/efficiency/payload/PayloadPoolMediator.hpp

```diff
@@ -119,10 +119,10 @@ class PayloadPoolMediator : public fastrtps::rtps::IPayloadPool
      * @return false if something went wrong
      */
     DDSPIPE_CORE_DllAPI
-    utils::ReturnCode write(
+    fastdds::dds::ReturnCode_t write(
```

Comment:
Piggyback request 3: restore the "RETCODE_" prefix to avoid collision problems.

Reply:
1. I added the missing value `ReturnCode::RETCODE_NOT_ENABLED` to the map
2. By declaring operator<< as a friend of the `ReturnCode` class we will have access to the protected member `to_string_conversion_`
3. `RETCODE_` prefix has been added to avoid collision problems